test/unit/MessageComposer/attachmentManager.test.ts

```diff
@@ -575,6 +576,35 @@ describe('AttachmentManager', () => {
   });
 
   describe('getUploadConfigCheck', () => {
+    it('should not check for file extension config if its a Blob type', async () => {
+      const blob = new Blob([''], { type: 'image/gif' });
+      const {
+        messageComposer: { attachmentManager },
+      } = setup();
+      const result = await attachmentManager.getUploadConfigCheck(blob);
+      expect(result).not.toEqual({
```

Comment:
reason should not be present if upload is not blocked. This assertion probably does not make sense

Reply:
You are right. I have removed it